sormas-ui/src/main/java/de/symeda/sormas/ui/person/PersonEditForm.java

```diff
@@ -403,6 +408,18 @@ protected void addFields() {
 				false,
 				I18nProperties.getValidationError(Validations.afterDate, burialDate.getCaption(), deathDate.getCaption())));
 
+		phoneNumber.addValidator(
```

Comment:
The PhoneNumberValidator we've written isn't really a good idea here; its main purpose is to ensure that the phone number is formatted like an international phone number which is necessary for the phone number added to users because those are then used for automatic SMS notifications. In retrospection, we should've probably named that class "UserPhoneNumberValidation" to make clear that it has a pretty small scope of usage.

For phone numbers of persons, we shouldn't enforce this. Instead, we should make sure that the phone number doesn't include characters that are invalid in phone numbers (e.g. letters I suppose). This isn't trivial because we, e.g., can't forbid characters like "(", ")", "-", "/" etc. because those are commonly used in different countries to format phone numbers.

We haven't clearly defined this yet which is why the issue was in the product backlog and not really "ready for development". If you have a proposal on how to do proper phone number validation, I'd like to hear it.

Reply:
@MateStrysewske thanks for the clarification, I think we can try one of these two things.

1. We can extend the existing PhoneNumberValidator to exclude invalid characters like letters and maybe other common things that phone numbers shouldn't have.
 
1. Write a new PhoneNumberValidator that can at least cover common validation formats for countries that Sormas supports, I know this process will be a long short, but we can pull it off.

Reply:
The first approach would only be possible if there's an easy way to "configure" the validator to ignore the existing rule that phone numbers have to start with a "+" sign for all fields except the phone number of a user.

In general, I think having some basic validation (e.g. no letters) is enough for now. Explicitely covering the validation formats of specific countries is a very volatile approach because as SORMAS is an Open Source project, we technically don't even know which countries are using it.